pulsar-broker/src/main/java/org/apache/pulsar/broker/web/PulsarWebResource.java

```diff
@@ -717,19 +717,19 @@ public CompletableFuture<Void> validateBundleOwnershipAsync(NamespaceBundle bund
                         throw new RestException(Status.PRECONDITION_FAILED,
                                 "Failed to find ownership for ServiceUnit:" + bundle.toString());
                     }
-                    // If the load manager is extensible load manager, we don't need check the authoritative.
-                    if (ExtensibleLoadManagerImpl.isLoadManagerExtensionEnabled(config())) {
-                        return CompletableFuture.completedFuture(null);
-                    }
                     return nsService.isServiceUnitOwnedAsync(bundle)
                             .thenAccept(owned -> {
                                 if (!owned) {
                                     boolean newAuthoritative = this.isLeaderBroker();
                                     // Replace the host and port of the current request and redirect
-                                    URI redirect = UriBuilder.fromUri(uri.getRequestUri()).host(webUrl.get().getHost())
-                                            .port(webUrl.get().getPort()).replaceQueryParam("authoritative",
-                                                    newAuthoritative).replaceQueryParam("destinationBroker",
-                                                    null).build();
+                                    UriBuilder uriBuilder = UriBuilder.fromUri(uri.getRequestUri())
+                                            .host(webUrl.get().getHost())
+                                            .port(webUrl.get().getPort())
+                                            .replaceQueryParam("authoritative", newAuthoritative);
+                                    if (!ExtensibleLoadManagerImpl.isLoadManagerExtensionEnabled(config())) {
```

Comment:
Do we need to add this condition?

Reply:
Yes, for the extensible load manager, the transfer operation needs to use `destinationBroker` as the destination broker.

Reply:
Oh, got it, thanks.

Reply:
Before, the `destinationBroker` is set in the leader broker's `ModularLoadManagerImpl#bundleBrokerAffinityMap`, so it requires redirecting to the leader and setting the `destinationBroker` before doing in the real unload operation.

The unload operation requires execution in the owner broker, which will redirect to the owner broker after setting the `destinationBroker` success. If we don't clean the `destinationBroker` parameter, it will fall into a loop.

The `ExtensibleLoadManagerImpl` used a different way to transfer the ownership to `destinationBroker`, so we do not need to clean the `destinationBroker` parameter when redirecting.

Reply:
nit: ppl will appreciate it if you can add these details as a comment.

Reply:
Got it, thanks.